packages/react-tinacms-github/src/github-client/github-client.ts

```diff
@@ -311,6 +315,30 @@ export class GithubClient {
     })
   }
 
+  async upload(
+    path: string,
+    fileContents: string,
+    commitMessage: string = 'Update from TinaCMS',
+    encoded: boolean = false
+  ) {
+    return this.githubFileApi(path, fileContents, commitMessage, encoded, 'PUT')
+  }
+
+  async delete(
+    path: string,
+    fileContents: string,
+    commitMessage: string = 'Update from TinaCMS',
+    encoded: boolean = false
+  ) {
+    return this.githubFileApi(
+      path,
+      fileContents,
+      commitMessage,
```

Comment:
Maybe we could have a more informative default commit message like:

```
`Deleted ${path} using TinaCMS`
```

Reply:
:+1:

Reply:
@ncphillips  Fixed